netstandard/ref/System.Data.cs

```diff
@@ -963,7 +963,7 @@ public sealed partial class DBConcurrencyException : System.SystemException
         public int RowCount { get { throw null; } }
         public void CopyToRows(System.Data.DataRow[] array) { }
         public void CopyToRows(System.Data.DataRow[] array, int arrayIndex) { }
-        public override void GetObjectData(System.Runtime.Serialization.SerializationInfo si, System.Runtime.Serialization.StreamingContext context) { }
+        public override void GetObjectData(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
```

Comment:
In https://github.com/dotnet/standard/pull/826 it was discussed if such a change is breaking or not. Was this discussion continued elsewhere or are renames of overrides now considered "safe"? (considering you can no longer call `new DBConcurrencyException().GetObjectData(si: null, context: default)` - not that I believe anyone has ever done so and I personally like the change)

Reply:
I vaguely remember the discussion ending with "It's technically a breaking change, but we're going to deem it not-sufficiently-breaking-to-worry-about-if-necessary." We should avoid breaking things like this unless there's a good reason, but it can be the "lesser evil" in many cases.

I think it would be worth detecting and reporting these "technically breaking changes" as part of the release notes though.